lib/datahub-client/data_platform_catalogue/client/graphql_helpers.py

```diff
@@ -32,32 +39,78 @@ def get_graphql_query(graphql_query_file_name: str) -> str:
     return query_text
 
 
-def parse_owner(entity: dict[str, Any]) -> OwnerRef:
+def _parse_owner(owner: dict):
+    properties = owner.get("properties") or {}
+    display_name = (
+        properties.get("displayName")
+        if properties.get("displayName") is not None
+        else properties.get("fullName", "")
+    )
+    return OwnerRef(
+        display_name=display_name or "",
+        email=properties.get(
+            "email",
+            _make_user_email_from_urn(owner.get("urn")),
+        ),
+        urn=owner.get("urn", ""),
+    )
+
+
+def parse_owner(
```

Comment:
We now have `parse_owner`, `_parse_owner`, and `parse_owners`, can we make do with just the plural? This might be satisfied by a function name change

Reply:
`parse_owner` is still used for the data owner, which I've left in the data model for now. The internal function I can rename though

Reply:
Maybe I can replace `parse_owner` with `parse_data_owner` to avoid confusion

Reply:
Reading the rest of the code here's a more thought out solution:
`parse_owner` -> `parse_data_owner`
`_parse_owner` -> `parse_generic_owner`